src/actions/tests/project-text-search.spec.js

```diff
@@ -101,4 +101,17 @@ describe("project text search", () => {
 
     expect(getTextSearchResults(getState())).toMatchSnapshot();
   });
+
+  it("should close project search", () => {
+    const { dispatch, getState } = createStore(threadClient);
```

Comment:
hmm, this test is almost there but it isn't testing what we need to test. we need to have a populated project search before closing it to determine of the close function is working properly. otherwise we are testing the default behavior (query is empty by default, results are empty by default). Maybe we can do this for now by dispatching a search query and mocking out the function that populates the list?

Reply:
ok, will update the test spec. 👍